src/goto-symex/goto_symex.h

```diff
@@ -463,6 +460,10 @@ class goto_symext
   }
 };
 
+nondet_symbol_exprt build_symex_nondet(typet &type, unsigned &nondet_count);
```

Comment:
Why is this an improvement?

Reply:
Better `static`?

Reply:
It's an improvement because the type makes it clear what is the data used and modified by this function, while before it could have effect on any of the class members.
I'm not sure it can be static because it's used in several cpp files.